fp.pretty/TDFN-8_B2.0x3.0x0.75mm_P0.5.kicad_mod

```diff
@@ -0,0 +1,38 @@
+(module TDFN-8_B2.0x3.0x0.75mm_P0.5 (layer F.Cu) (tedit 5F3D1648)
+  (attr smd)
+  (fp_text reference REF** (at -2.375 0 90) (layer F.SilkS)
+    (effects (font (size 1 1) (thickness 0.15)))
+  )
+  (fp_text value TDFN-8_B2.0x3.0x0.75mm_P0.5 (at 0.52 -3.06) (layer F.Fab)
+    (effects (font (size 1 1) (thickness 0.15)))
+  )
+  (fp_line (start -0.575 2.2) (end -1.375 1.4) (layer F.SilkS) (width 0.12))
+  (fp_line (start -1.375 1.4) (end -1.375 -2.2) (layer F.SilkS) (width 0.12))
+  (fp_line (start -1.375 -2.2) (end 1.375 -2.2) (layer F.SilkS) (width 0.12))
+  (fp_line (start 1.375 -2.2) (end 1.375 2.2) (layer F.SilkS) (width 0.12))
+  (fp_line (start 1.375 2.2) (end -0.575 2.2) (layer F.SilkS) (width 0.12))
+  (fp_line (start -1.13 -1.95) (end 1.13 -1.95) (layer F.CrtYd) (width 0.05))
+  (fp_line (start 1.13 -1.95) (end 1.13 1.95) (layer F.CrtYd) (width 0.05))
+  (fp_line (start 1.13 1.95) (end -1.13 1.95) (layer F.CrtYd) (width 0.05))
+  (fp_line (start -1.13 1.95) (end -1.13 -1.95) (layer F.CrtYd) (width 0.05))
+  (fp_text user REF** (at 0.11 -0.14 90) (layer F.Fab)
+    (effects (font (size 0.75 0.75) (thickness 0.15)))
+  )
+  (fp_line (start -1.25 -2.06) (end 1.24 -2.06) (layer F.Fab) (width 0.12))
+  (fp_line (start 1.24 -2.06) (end 1.24 2.05) (layer F.Fab) (width 0.12))
+  (fp_line (start 1.24 2.05) (end -1.25 2.05) (layer F.Fab) (width 0.12))
+  (fp_line (start -1.25 2.05) (end -1.24 -2.05) (layer F.Fab) (width 0.12))
+  (pad 8 smd rect (at -0.75 -1.5) (size 0.25 0.4) (layers F.Cu F.Paste F.Mask))
+  (pad 1 smd rect (at -0.75 1.5) (size 0.25 0.4) (layers F.Cu F.Paste F.Mask))
+  (pad 7 smd rect (at -0.25 -1.5) (size 0.25 0.4) (layers F.Cu F.Paste F.Mask))
+  (pad 2 smd rect (at -0.25 1.5) (size 0.25 0.4) (layers F.Cu F.Paste F.Mask))
+  (pad 6 smd rect (at 0.25 -1.5) (size 0.25 0.4) (layers F.Cu F.Paste F.Mask))
+  (pad 3 smd rect (at 0.25 1.5) (size 0.25 0.4) (layers F.Cu F.Paste F.Mask))
+  (pad 5 smd rect (at 0.75 -1.5) (size 0.25 0.4) (layers F.Cu F.Paste F.Mask))
+  (pad 4 smd rect (at 0.75 1.5) (size 0.25 0.4) (layers F.Cu F.Paste F.Mask))
+  (model "${KIPRJMOD}/3D Models/User Library-TDFN-8-1.STEP"
```

Comment:
This should be a path in `${KISYS3DMOD}`, see here: https://kicad-pcb.org/libraries/klc/F9.3/

Reply:
Done

Reply:
Done